pages/BasePage.js

```diff
@@ -0,0 +1,42 @@
+const PAGE_TIMEOUT = 10000
+export default class BasePage {
+
+  /**
+   * Abrir página
+   * @param {String} ruta a la cual acceder
+   */
+  async abrir(ruta) {
+      await browser.url(`${ruta}`);
```

Comment:
if the string you are building has only the parameter you are receiving for the method, there's no need for using **template literals**. You can simplify it to just:

```javascript
  async open(route) {
      await browser.url(route);
}
```